stdlib/experimental/experimental.flux

```diff
@@ -1252,3 +1252,34 @@ builtin histogram : (
         bins: [float],
         ?normalize: bool,
     ) => stream[{T with _value: float, le: float}]
+
+// preview limits the number of rows and tables in the stream.
+// The group keys that are included are not deterministic and depends on the order
+// that the engine sends them.
+//
+// ## Parameters
+// - nrows: Maximum number of rows in each table.
+//
+// - ntables: Maximum number of tables returned by the transformation.
+//
+// - tables: Input data. Default is piped-forward data (`<-`).
+//
+// ## Examples
+//
+// ### Preview data output
+// ```
+// import "experimental"
+// import "sampledata"
+//
+// sampledata.init()
+//   |> experimental.preview()
+//
+// ## Metadata
+// introduced: next
```

Comment:
@nathanielc Does case matter here? Does this need to be `NEXT`?

Reply:
It looks like it's supposed to be `NEXT`. Fixed.